src/ngraph_sgcompiler.cc

```diff
@@ -435,6 +435,8 @@ void SGCompiler::CompileNodes(NodePtr node,
         this->op_map_.insert(
             {node, this->ngraph_op_funcs_[node->operation_](node)});
 
+        if (node->multi_output_index_ == -1) return;
```

Comment:
Would it be okay to add curly braces around the `return` statement?

IME, using curly braces even around single-statement code blocks reduces the risk of certain hard-to-debug problems introduced by later code changes.

Reply:
agree, i try to use braces around following ngraph repo, missed this, will fix.